api/models/token.go

```diff
@@ -0,0 +1,48 @@
+package models
+
+import (
+	"errors"
+	"time"
+
+	"github.com/PedroChaparro/loomies-backend/interfaces"
+	"github.com/golang-jwt/jwt/v4"
+)
+
+// secret key
+
+var JWTKey = []byte("my_secret_key")
+
+// create jwt, accessToken and refreshToken
+
+func CreateToken(info interfaces.TokenInfo) (string, string, *jwt.Token, *jwt.Token, error) {
```

Comment:
1. We don't really need to return the *jwt.Token, the signed string should be OK.

2. I think it would be better to separate this function into: `CreateAccessToken` and `CreateRefreshToken`, however, i don't think we need a refreshToken.